features/gba-dev/uitsluiting-kiesrecht.feature

```diff
@@ -0,0 +1,20 @@
+# language: nl
+
+Functionaliteit: Uitsluiting kiesrecht
+
+
+	@gba
+	Abstract Scenario: Geen aanduiding uitgesloten kiesrecht (38.10) wel een einddatum uitsluiting kiesrecht
+	  Gegeven het systeem heeft een persoon met de volgende gegevens
+		| naam                        | waarde     |
+		| burgerservicenummer         | 999990001  |
+    En de persoon heeft de volgende kiesrecht gegevens
+    | einddatum uitsluiting kiesrecht (38.20) | 20300101 |
+		Als de persoon op 15 maart 2022 wordt geraadpleegd met de volgende parameters
```

Comment:
Zoals aangegeven in #1223 is deze stap niet te automaten. De systeem tijd in de gba api container moet worden aangepast. Is er een reden om `op 15 maart 2022` in de stap te laten staan?

Reply:
meegekomen met kopiëren van scenario. Heb gewijzigd naar "Als personen wordt gezocht met de volgende parameters". Scenario zal werken tot 2030.